src/components/Projects/Projects.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { BlogCard, CardInfo, ExternalLinks, GridContainer, HeaderThree, Hr, Tag, TagList, TitleContent, UtilityList, Img } from './ProjectsStyles';
+import { Section, SectionDivider, SectionTitle } from '../../styles/GlobalComponents';
+import { projects } from '../../constants/constants';
```

Comment:
This is a good way to do this, so you can have some component reuse. You could also use a `.json` file but it doesn't really matter.